### ISSUE TITLE
The function of historical forecast

### ISSUE THREAD

**Issue by pikawangwang** [unit8co/darts#2347]
I am a student trying to do time series forecasting.

When I am trying to using the function historical forecast with training set, I am confused about how the training loss is generated.

Is there a default fold of cross validaion, or the loss is generated by all of the training set?

Thank you！

**Comment by madtoinou** [comment]
Hi @pikawangwang,

You can find this information in the docstring of `historical_forecasts()` : "This method repeatedly builds a training set: either expanding from the beginning of series or moving with a fixed length `train_length`. It trains the model on the training set, emits a forecast of length equal to forecast_horizon, and then moves the end of the training set forward by `stride` time steps.

By default, this method will return one (or a sequence of) single time series made up of the last point of each historical forecast. This time series will thus have a frequency of series.freq * stride. If `last_points_only` is set to False, it will instead return one (or a sequence of) list of the historical forecasts series."

This method does not compute any loss/metric, but generate forecasts as if the model was fitted and called "from the past", to mimic historic/real-world usage of the model. Which can then be used to assess the model performance.

Does it make things clearer?

**Comment by pikawangwang** [comment]
@madtoinou Thanks for your explanation, I understand now😀
